shared-bindings/displayio/Display.h

```diff
@@ -59,6 +59,9 @@ bool displayio_display_send_pixels(displayio_display_obj_t* self, uint32_t* pixe
 bool common_hal_displayio_display_get_auto_brightness(displayio_display_obj_t* self);
 void common_hal_displayio_display_set_auto_brightness(displayio_display_obj_t* self, bool auto_brightness);
 
+mp_int_t common_hal_displayio_display_get_width(displayio_display_obj_t* self);
```

Comment:
Use `uint16_t` here instead of `mp_int_t`, because that's the type of width and height that are passed in.